.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: node_js
 node_js:
-  - "4.0.0"
-  - "5"
+  - "6.0.0"
   - "6"
```

Comment:
You're specifying both `"6"` and `"6.0.0"` because the latter is the oldest version you support?

Reply:
Correct.